package.json

```diff
@@ -7,6 +7,7 @@
     "@navikt/aksel-icons": "3.4.2",
     "@tanstack/react-query": "4.29.12",
     "@tanstack/react-query-devtools": "4.29.12",
+    "autosize": "^6.0.1",
```

Comment:
Vi bør være forsiktige med å dra inn avhengigheter fra tilfeldige privatpersoner. Dette er kode vi ikke har kontroll på. Har du vurdert dette mot andre alternativer? Kunne vi prøvd å lage denne funksjonaliteten selv, eller blir det for komplisert?
Hvis vi får til å lage dette uten avhengigheter, tenker jeg at vi bør få det inn i designsystemet.